android/OnesignalPlugin.java

```diff
@@ -68,6 +69,7 @@ public void onCreate(Activity activity, Bundle savedInstanceState) {
           OneSignal.init(activity, g_Project_Number, appID, new OpenedHandler(), new ReceivedHandler());
           OneSignal.setInFocusDisplaying(OneSignal.OSInFocusDisplayOption.Notification);
           onesignal = true;
+          ShortcutBadger.applyCount(activity, 0);
```

Comment:
i dont think so
https://github.com/leolin310148/ShortcutBadger/blob/master/ShortcutBadger/src/main/java/me/leolin/shortcutbadger/ShortcutBadger.java line 62 ,it already put it in try catch and log also